src/routes/index.svelte

```diff
@@ -11,9 +15,14 @@
 	<title>Challenge Bombs</title>
 </svelte:head>
 <h1 class="header">Challenge Bombs</h1>
+<HomeSearchBar missions={missions}
+	missionCards={missionCards}>
+</HomeSearchBar>
 <div class="bombs">
-	{#each missions as mission}
-		<MissionCard {mission} />
+	{#each missions as mission, index}
+		<MissionCard mission={mission} 
```

Comment:
The "svelte" way to do this would be to wrap the MissionCard in an `{#if ...}` or setting a class depending on if that mission matches the filter. Instead binding the cards and then toggling classes from the LayoutSearchFilter. Refactoring it to work that way is something that you can do only if you really want. At least another way to look at the problem in the future.

Reply:
I don't see how I could do that and still have the layers of abstraction. The LayoutSearchFilter is designed to be able to hide **any** array or grid of items based on the filter function called by the parent component. So it can't possibly know about the mission card array since that's a different layer of abstraction.